src/ssl.c

```diff
@@ -58108,7 +58108,8 @@ int wolfSSL_CONF_cmd(WOLFSSL_CONF_CTX* cctx, const char* cmd, const char* value)
             bio->method = method;
 #endif
             bio->shutdown = BIO_CLOSE; /* default to close things */
-            bio->num = WOLFSSL_BIO_ERROR;
+            bio->len = WOLFSSL_BIO_ERROR;
```

Comment:
Doesn't look like the right value to set for a length.

Reply:
Changed to `0`.